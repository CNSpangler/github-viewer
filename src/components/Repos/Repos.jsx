import React from 'react';
import PropTypes from 'prop-types';
import Repo from '../Repo/Repo.jsx';

const Repos = ({ repos }) => (
  <ul>
    {repos.map(repo => {
      <Repo repo={repo} />;
    })}
  </ul>
);

Repos.propTypes = {
  repos: PropTypes.array.isRequired,
};

export default Repos;
