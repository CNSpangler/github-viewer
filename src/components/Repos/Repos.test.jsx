import React from 'react';
import { shallow } from 'enzyme';
import Repos from './Repos.jsx';

describe('Repos component', () => {
  it('renders Repos', () => {
    const wrapper = shallow(
      <Repos 
        name="portfolio"
        url="https://github.com/CNSpangler/portfolio"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
