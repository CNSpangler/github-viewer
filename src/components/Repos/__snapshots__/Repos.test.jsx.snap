// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repos component renders Repos 1`] = `
<ul>
  <li>
    <Repo
      html_url="https://github.com/CNSpangler/pokedex-router"
      name="pokedex-router"
    />
  </li>
  <li>
    <Repo
      name="portfolio"
      url="https://github.com/CNSpangler/portfolio"
    />
  </li>
</ul>
`;
